tools/packaging/guest-image/build_image.sh

```diff
@@ -39,13 +39,19 @@ build_initrd() {
 	export OS_VERSION="${initrd_os_version}"
 	export USE_DOCKER=1
 	export AGENT_INIT="yes"
+	config_version=$(get_config_version)
 	# ROOTFS_BUILD_DEST is a Make variable
 	# SNP will also use the SEV guest module
 	if [[ "${AA_KBC:-}" == "offline_sev_kbc" || "${AA_KBC:-}" == "online_sev_kbc" ]]; then
-		config_version=$(get_config_version)
 		kernel_version="$(get_from_kata_deps "assets.kernel.sev.version")"
 		kernel_version=${kernel_version#v}
 		module_dir="${repo_root_dir}/tools/packaging/kata-deploy/local-build/build/cc-sev-kernel/builddir/kata-linux-${kernel_version}-${config_version}/lib/modules/${kernel_version}"
+	else
+		kernel_version="$(get_from_kata_deps "assets.kernel.version")"
+		kernel_version=${kernel_version#v}
+		module_dir="${repo_root_dir}/tools/packaging/kata-deploy/local-build/build/cc-kernel/builddir/kata-linux-${kernel_version}-${config_version}/lib/modules/${kernel_version}"
+	fi
+	if [ "${KATA_BUILD_CC}" == "yes" ]; then
```

Comment:
This part here will cause a behaviour change, as we'll start trying to add kernel modules even for the targets that do not use it (TDX and vanilla).

Have you done some tests to check whether this will cause any regression?

Reply:
yep, I've tested newly built kernel, image and initrd without errors. But it is true that those additional built modules are only needed for cc-initrd. That is why I left `harsh review` for better solution. Thanks.

Reply:
If SE (secure execution for IBM Z) requires some special modules like SEV, we could make a dedicated target like `cc-se-kernel-tarball`, but the thing is that a common initrd for CC does not work without those crypto modules.

Reply:
Or why don't we make a dummy target which is only used for cc-initrd?

Reply:
Any chance this kernel module is only needed for offline_fs_kbc and it could also delineate based on AA_KBC?

Reply:
I decided to go with a SE specific dummy kernel build. The regression should be successful. (Please check my recent brand new changes)